bindings/python/pymongocrypt/explicit_encrypter.py

```diff
@@ -194,19 +207,26 @@ def encrypt(self, value, algorithm, key_id=None, key_alt_name=None,
           - `query_type` (str): The query type to execute.
           - `contention_factor` (int): The contention factor to use
             when the algorithm is "Indexed".
+          - `range_options` (bytes): Options for explicit encryption
+            with the "rangePreview" algorithm encoded as a BSON document.
+          - `encrypt_expression` (boolean): True if this is an
```

Comment:
This sounds like it should be the expression itself.  Perhaps `is_encrypt_expression`?